### ISSUE TITLE
PDF files larger than expected when subsetting fonts

### ISSUE THREAD

**Issue by michaelrsweet** [apple/cups#789]
Version: 7.07.1
CUPS.org User: jasondotaikenatmedtronic

I received the attached message in response to STR 721. However, I am unable to add comments or update any information at STR 721.

The report I submitted shows that 60% of the PDF file size is font information. The font subsets should be handled differently. Can you re-consider this issue from that perspective?

Regards,
Jason

p.s. I'm sorry that I brought AFPL GS 8.00 into the discussion. 

Software trouble report #721 has been updated.  You can check
the status of the report and add additional comments and/or files
at the following URL:

```
http://www.cups.org/str.php?L721

Summary: PDF files larger than expected when subsetting fonts
Version: ESP GS 7.07.1
 Status: 2 - Closed w/o Resolution
```

   Priority: 2 - Low, e.g. a documentation error or undocumented side-effect
      Scope: 3 - Applies to all machines, printers, and operating systems
  Subsystem: ESP Ghostscript
Fix Version: Unassigned

AFPL Ghostscript 8.x is SIGNIFICANTLY different than 7.07, and there is
nothing we can do without backporting the 8.x code, which is provided
under a different license.

In short, this will not be fixed until Artifex releases 8.0 as GPL and we
provide a version of ESP Ghostscript based on that release.
